requirements.txt

```diff
@@ -67,3 +67,4 @@ httplib2~=0.9.2
 confluent-kafka==1.0.0
 unidecode
 boto3==1.10.1
+cassandra-driver==3.20.1
```

Comment:
cc @jinhyukchang 
I think we could put it in requirements.txt for now. In long run, we should make it as optional deps and put it in setup.py.

Reply:
I was just thinking about this after my glue extractor PR, most libs add the code even without the requirements and do a try catch on the import to warn the user to install

Reply:
that sounds good as well :)